lib/decrypt.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'enigma'
+require_relative 'shift'
+
+enigma_machine = Enigma.new
+
+encrypted_reader = File.open(ARGV[0], "r")
+
+encrypted_message = encrypted_reader.read
+
+encrypted_reader.close
+
+decrypted_message = enigma_machine.decrypt(encrypted_message, ARGV[2], ARGV[3])
```

Comment:
used ARGV directly in arg, before I tried to set them to a variable then call that variable but it wasn't working for some reason.